lib/rdf/ldp/helper.rb

```diff
@@ -3,14 +3,16 @@
 # mixin methods
 module RDF
   class LDP < RDF::StrictVocabulary("http://www.w3.org/ns/ldp")
+    extend Deprecation
```

Comment:
I think neither - I think I was incorrect in putting it there at all.  See deprecation gem and for examples, you can look in projectblacklight/blacklight.